### PR TITLE
Pipenv

### PR DESCRIPTION
**Pull request by lsh-0**
I was getting some deprecation warnings in Python3.6 and Python3.8 and a version conflict with boto with elife-spectrum.

Is `econ-tools` a library or more of a standalone app?

* Adds a Pipfile and a Pipfile.lock
* updates requirements.txt, prunes some unused dependencies
* updates the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,43 +16,32 @@ You must set the following environment variables before running the program:
...
-    
-This environment should be activated when running any of the tools, e.g.
-
-    source venv/bin/activate
-
 ## econ_dashprop
 
-econ_dashboard string properties against versions of articles within the eLife Continuum Dashboard.
-An example use for this is updating the publication-status property of an article to indicate it should be temporarily hidden.
+`econ_dashboard` string properties against versions of articles within the eLife Continuum Dashboard.
```
Comment (lsh-0) [inline README.md:21]: this doesn't make sense to me

File: update-dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# create/update existing venv
+rm -rf venv/
+
+# whatever your preferred version of python is, eLife needs to support python3.6 (Ubuntu 18.04)
+python3.6 -m venv venv
+
+# prefer using wheels to compilation
+source venv/bin/activate
+pip install wheel
```
Comment (lsh-0) [inline update-dependencies.sh:13]: I could probably get rid of this, only useful on projects where binaries are needed

Comment (gnott) [reply update-dependencies.sh:13]: I remember using that to create alternate packages for pypi, but I'm not sure if it's need here, or needed anymore.

File: README.md (hunk 1 above)
Comment (gnott) [reply README.md:21]: I think the second line refers to the example below it, the `publication-status hidden` results in sending the `publication-status` value of `hidden` onto the `event-property-incoming-queue` for article `00288`, version `1`. That message is then picked up by the dashboard, which results in hiding version 1 of `00288` from the current list of articles that are errored or in progress. I do this occassionally for Production, and I use the example from the readme to remind me how to do it. The wording probably doesn't make sense unless you already know what it's describing. Are you able to improve it for better understanding, if I explained it better in the above?

**Review by gnott — APPROVED** [review]
Ok by me, we can adjust as time goes on for Python > 3.6 etc. when applicable

File: update-dependencies.sh (hunk 2 above)
Comment (lsh-0) [reply update-dependencies.sh:13]: it's needed in some projects actually. Without it, some things like `cryptography` try to compile their own on the system and usually fail unless a bunch of dev dependencies are there (like `python-dev`, `libpq` etc). I thought simply having it installed on the system was enough but it needs to be there in the virtualenv itself. One of those lessons that was more frustrating than it should have been.

File: README.md (hunk 1 above)
Comment (lsh-0) [reply README.md]: just took a look at the code and something was lost in the copy+paste to the readme it seems. It makes more sense now.

Thanks for the explanation, @gnott , I've added it beneath the example

**Pull request merged**